cabal-install/src/Distribution/Client/ScriptUtils.hs

```diff
@@ -377,7 +377,7 @@ withContextAndSelectors noTargets kind flags@NixStyleFlags{..} targetStrings glo
 
               executable' =
                 executable
-                  & L.buildInfo . L.defaultLanguage %~ maybe (Just Haskell2010) Just
```

Comment:
What about here @ffaf1 ?

Reply:
Here I tested in the same fashion as above. The function is `withContextAndSelectors` and it is used by — among others — `build`.

`-v3` does not show any `Haskell2010`, only `Haskell98`.

With *scripts* (`cabal run -v3 prova.hs`), I am trying now... and indeed they differ. Master has `Haskell2010`, my branch `Haskell98`.

Well spotted @alt-romes , many thanks.

Reply:
It's not entirely clear, but my stance here would be: document it as is, don't break anything.

Reply:
That is, from what I've understood: Haskell98 is used by default on cabal package components without a default-language field. However, cabal scripts without a default language will use Haskell2010 by default instead?

Reply:
That seems to be the case.

If we document it this way (which is correct), we also need to come up with tests, because we are one swapping `a <> b` with `b <> a` from breakage.